python/servo/post_build_commands.py

```diff
@@ -95,8 +95,18 @@ def run(self, params, release=False, dev=False, android=False, debug=False, debu
                 print("Could not find a suitable debugger in your PATH.")
                 return 1
 
+            command = self.debuggerInfo.path
+            if debugger == 'gdb' or debugger == 'lldb':
+                rustCommand = 'rust-' + debugger
+                try:
+                    result = subprocess.call([rustCommand, '--version'], env=env, stdout=open(os.devnull, 'w'))
```

Comment:
Argh, I'm confusing myself: of course the second point is also still valid when using `which` vs. attempting to call the actual command, regardless of the call() variant used.